planner/core/plan_cost.go

```diff
@@ -26,15 +26,27 @@ import (
 	"github.com/pingcap/tidb/util/paging"
 )
 
+const (
+	// CostFlagRecalculate indicates the optimizer to ignore cached cost and recalculate it again.
+	CostFlagRecalculate uint64 = 1 << iota
+
+	// CostFlagUseTrueCardinality indicates the optimizer to use true cardinality to calculate the cost.
+	CostFlagUseTrueCardinality
```

Comment:
In the next PR, I'll use these two flags to support `explain analyze format='true_card_cost' ...`.